tools/create-ec2-rootfs.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+#
+# Copyright Amazon.com Inc. or its affiliates.
```

Comment:
Hi Kuniyuki,

Is it possible to re-license under the standard project license?
If yes, please do, need it to merge.
If not, then you can upload it somewhere else and we can reference it from syzkaller repo somewhere.

Reply:
Hi Dmitry,

Yes, I'll update it like this.

```
# Copyright 2023 syzkaller project authors. All rights reserved.
# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
```